features/fixtures/ios-swift-cocoapods/iOSTestApp/scenarios/ReadGarbagePointerScenario.m

```diff
@@ -32,6 +32,11 @@
  */
 @implementation ReadGarbagePointerScenario
 
+- (void)startBugsnag {
+    self.config.shouldAutoCaptureSessions = NO;
```

Comment:
Is it worth making this the default in the mazerunner fixture's AppDelegate, seeing as most of the scenarios already assume this?

Reply:
No, because having it there made it difficult to test the default behavior for session tracking. For example, the auto-capture scenarios should test the behavior when not setting the value at all to ensure the default behavior is to send a session request. It’s also easier to debug an individual case when only setting the minimal values for running the scenario by default. I lost a bit of time to confusion when I couldn’t figure out where a config value was being set between AppDelegate, Scenario, and the Scenario subclass.